README.md

```diff
@@ -1,2 +0,0 @@
-# dynamodb-sink
```

Comment:
Replaced the README.md with README.rst. Should we have both the md and rst files in the repo? Or only having README.rst is okay?